src/Salt/SiteBundle/Entity/Comment.php

```diff
@@ -64,7 +64,10 @@ class Comment
     /**
      * @var string
      *
-     * @ORM\Column(type="string")
+     * @Serializer\Accessor(getter="getFullname")
+     * @Serializer\ReadOnly
+     * @Serializer\Type("string")
+     * @Serializer\SerializedName("fullname")
      */
     private $fullname;
```

Comment:
There is no need for this field anymore, the `Serializer` annotations can be moved to be a [virtual property](http://jmsyst.com/libs/serializer/master/reference/annotations#virtualproperty).